aana/integrations/external/yt_dlp.py

```diff
@@ -10,6 +11,42 @@
     DownloadException,
 )
 
+class VideoInfo(TypedDict):
+    """Represents a video title, description and duration."""
+    title: str
+    description: str
+    duration: float
+
+def get_video_metadata(video_url: str) -> VideoInfo:
```

Comment:
Should it be video_url or video input object?

Reply:
I thought of that.
It will work just for the video serving sites like youtube, so I keep it to accept a url, the user would handle it itslef. because most of the time it will be an empty output.